src/Sentry.Unity.Editor/Native/BuildPostProcess.cs

```diff
@@ -148,14 +168,24 @@ private static void UploadDebugSymbols(IDiagnosticLogger logger, BuildTarget tar
                 if (isMono)
                 {
                     addPath(Path.GetFileNameWithoutExtension(executableName) + "_Data/MonoBleedingEdge/x86_64");
+                    addFilesMatching(buildOutputDir, new[] { "*.debug" });
                 }
             }
             else if (target is BuildTarget.StandaloneOSX)
             {
                 addPath(Path.GetFullPath($"Packages/{SentryPackageInfo.GetName()}/Plugins/macOS/Sentry/Sentry.dylib.dSYM"));
             }
 
-            var cliArgs = "upload-dif --il2cpp-mapping ";
+            if (isMono)
+            {
+                addFilesMatching($"{projectDir}/Temp", new[] { "**/UnityEngine.*.pdb", "**/Assembly-CSharp.pdb" });
```

Comment:
Do you know by any chance if that is true for newer Unity versions as well? I lost track of what Unity caches where in which version.